ansible_rulebook/websocket.py

```diff
@@ -209,12 +226,26 @@ async def _handle_request_workload(
             response.variables = yaml.safe_load(
                 base64.b64decode(data.get("data"))
             )
+        if data.get("type") == "EnvVars":
+            response.env_vars = yaml.safe_load(
+                base64.b64decode(data.get("data"))
+            )
         if data.get("type") == "ControllerInfo":
             response.controller_url = data.get("url")
             response.controller_token = data.get("token")
             response.controller_ssl_verify = data.get("ssl_verify")
             response.controller_username = data.get("username", "")
             response.controller_password = data.get("password", "")
+
+    if non_fq_key and "filename" in file_template_vars:
+        response.variables["eda"] = {
+            "filename": file_template_vars["filename"]
+        }
+    else:
+        response.variables["eda"] = {"filename": file_template_vars}
```

Comment:
@bzwei These issues have to trapped in the eda-server side.
We need a JIRA to prevent users from adding a variable called eda and ansible.eda. Currently the eda-server allows any field name.
The same can be said about mixing template and template.key_file use case